beacon-chain/endtoend/evaluators.go

```diff
@@ -0,0 +1,146 @@
+package endtoend
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	ptypes "github.com/gogo/protobuf/types"
+	"github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/shared/params"
+)
+
+// Formally defining them might not be needed.
+// type policy func(chainHead *eth.ChainHead, options ...uint64) error
+// type evaluation func(client *eth.BeaconChainClient, options ...uint64) error
+
+// // Evaluator defines the function signature for function to run during the E2E.
+// type Evaluator struct {
+// 	Policy     policy
+// 	Evaluation evaluation
+// }
+
+func RunChainStartEvaluators(t *testing.T, client eth.BeaconChainClient, currentEpoch uint64) {
+	if OnChainStart(currentEpoch) {
+		fmt.Println("Running chainstart test")
+		t.Run("validators activate", func(t *testing.T) {
```

Comment:
These runners typically don't have spaces formatting-wise, maybe we can opt for something more standard like
```
t.Run("activate_validators", ...
```